types/istanbuljs__nyc-config-babel/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for @istanbuljs/nyc-config-babel 3.0
+// Project: https://github.com/istanbuljs/istanbuljs/tree/master/packages/nyc-config-babel
+// Definitions by: Paul Sanchez <https://github.com/basicdays>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const config: {
+    sourceMap: boolean;
+    instrument: boolean;
+    require: string[];
```

Comment:
```suggestion
    require: ['@babel/register']
```
this could better describe with literal tuple, but it's optional change

Reply:
I considered making the types here literal values, but went with the direction of something slightly more generic with the theory that the config could change without having to change the types.